test/frame.sh

```diff
@@ -70,4 +70,4 @@ cat << 'EOF' > "$TMP/expected.txt"
     - (fragment)     : /$defs/string
 EOF
 
-diff "$TMP/result.txt" "$TMP/expected.txt"
```

Comment:
This looks like an unintended change

Reply:
It doesn't have any difference between the original version, still I do not know why it says there is a diff

Reply:
It's the final new line character at the end of the file. We saw it in other PRs. It's likely a misconfiguration on your editor. Google how to make sure your editor includes an end-of-file line ending

Reply:
See the symbol that GitHub shows on the diff:

<img width="329" alt="Screenshot 2024-07-11 at 11 54 41 PM" src="https://github.com/user-attachments/assets/774a883c-2212-43bf-8f74-74a04e22da3e">

Reply:
Shet! sorry I got it, I think myself I removed the EOF, I will correct it